theano/gof/tests/test_utils.py

```diff
@@ -79,8 +79,8 @@ def __init__(self, var):
     class CachingClassExample_3(theano.Op):
         __props__ = ()
 
-        def __init__(self, var):
-            self.var = var
+        def __init__(self):
+            pass
```

Comment:
I did implement this test at the time you asked. But these tests are not valid anymore as we have changed the implementation.